### PR TITLE
[UPGRADE] version 0.19.0 stable

### PR DESCRIPTION
**Pull request by dan-ss93**
Solves basecamp's: https://3.basecamp.com/4186608/buckets/12708820/todos/2143771617

Upgraded to version 0.19.0 stable.
- Updated decidim gem to get latest bugfix
- Changed term_customizer dependency to main repository

### CODE REVIEW

**Review by aitorlb — CHANGES_REQUESTED** [review]
Update `decidim` gem again to include latest bugfixes in `0.19-stable` branch

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ruby RUBY_VERSION
 
 # DECIDIM_VERSION = '~> 0.18.0'
 # We are pointing to this branch, because we need the new developments of the branch
```
Comment (aitorlb) [inline Gemfile]:
```suggestion
```

Comment (aitorlb) [inline Gemfile]: Remove these comments, please :point_up_2:

Comment (victorol1) [reply Gemfile]: Done

**Comment by victorol1** [comment]
> Update `decidim` gem again to include latest bugfixes in `0.19-stable` branch

Done!

**Pull request merged**